articles/azure/azs-how-configure-mars-backups-powershell.md

```diff
@@ -0,0 +1,91 @@
+---
+title: How to backup file(s) and folder(s) on Azure Stack VMs using Microsoft Azure recovery services (MARS) - PowerShell | UKCloud Ltd
+description: Provides details on how to install and configure the MARS agent to backup file(s) and folder(s) on Azure Stack
```

Comment:
Spell out MARS in full here as well as providing the acronym and use Init Caps for product names:
Provides details on how to install and configure the Microsoft Azure Recovery Services (MARS) agent to backup files and folders on Azure Stack